sdk/ml/azure-ai-ml/tests/component/e2etests/test_component.py

```diff
@@ -760,6 +760,7 @@ def test_simple_pipeline_component_create(self, client: MLClient, randstr: Calla
             "type": "pipeline",
         }
         assert component_dict == expected_dict
+        # below line is expected to raise KeyError in live test, it will pass after related changes deployed to canary
         jobs_dict = rest_pipeline_component._to_dict()["jobs"]
```

Comment:
then will you skip it in live mode for now or wait for the release?

Reply:
synced with Brynn, we will wait for the release, and let this test break now...